libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -38,19 +39,14 @@ data class NavigationOptions constructor(
     val onboardRouterConfig: MapboxOnboardRouterConfig?,
     val isFromNavigationUi: Boolean = false,
     val isDebugLoggingEnabled: Boolean = false,
-    val deviceProfile: DeviceProfile
+    val deviceProfile: DeviceProfile,
+    val builder: Builder
 ) {
 
     /**
      * Get a builder to customize a subset of current options.
      */
-    fun toBuilder() = Builder()
-        .accessToken(accessToken)
-        .timeFormatType(timeFormatType)
-        .navigatorPredictionMillis(navigatorPredictionMillis)
-        .distanceFormatter(distanceFormatter)
-        .onboardRouterConfig(onboardRouterConfig)
-        .isFromNavigationUi(isFromNavigationUi)
```

Comment:
notice that two values were not added here
1. deviceProfile
2. isDebugLoggingEnabled